lang/pl/content.php

```diff
@@ -23,41 +24,55 @@
             "title_2" => "zadowolonych klientów Blumilk",
         ],
         "section_3" => [
-            "title_1" => "Jak możemy Ci pomóc?",
-            "subtitle_1" => "Badania, rozwój, kod: ",
-            "subtitle_2" => "kształtujemy przyszłość oprogramowania.",
-            "content_1" => "Projekty B+R",
-            "content_2" => "Audyty rozwiązań IT",
-            "content_3" => "Cyfryzacja biznesu i procesów",
-            "content_4" => "Dedykowane aplikacje mobilne i webowe",
-            "title_2" => "Projektujemy rozwiązania",
-            "subtitle_3" => "W trosce o stuprocentowe zadowolenie naszych klientów i partnerów.",
+            "title_1" => "Co możemy dla Ciebie zrobić?",
+            "subtitle_1" => "Twój biznes potrzebuje transformacji cyfrowej? Jesteś we właściwym miejscu.",
+            "tile_1" => [
+                "title" => "Aplikacje mobilne i webowe",
+                "subtitle" => "Tworzymy dedykowane oprogramowanie w formie aplikacji mobilnych i webowych.",
+            ],
+            "tile_2" => [
+                "title" => "Audyty rozwiązań IT",
+                "subtitle" => "Analizujemy kody źródłowe gotowych rozwiązań informatycznych.",
+            ],
+            "tile_3" => [
+                "title" => "Cyfryzacja biznesu i procesów",
+                "subtitle" => "Pomagamy klientom przejść przez transformację cyfrową w ich firmach.",
+            ],
+            "tile_4" => [
+                "title" => "Projekty B+R",
+                "subtitle" => "Angażujemy się w szeroki zakres popularnych zagadnień badawczych.",
+            ],
+            "tile_5" => [
+                "title" => "Interface design",
```

Comment:
All titles we have in Polish, so why this is in English?

Reply:
Did the same as is designed

Reply:
I spoke with graphic designer and we can set there title "UI/UX"